src/components/custom/review/review.tsx

```diff
@@ -39,6 +45,40 @@ export const Review = (props: IGenericCustomElementProps<IReviewSchema>) => {
 	// =========================================================================
 	// RENDER FUNCTIONS
 	// =========================================================================
+	if (schema.variant === "accodion") {
+		const { items, topSection, bottomSection, button, title, expanded, collapsible, ...otherSchema } = schema;
+		return (
+			<BoxContainer
+				id={id}
+				data-testid={id}
+				title={title}
+				expanded={expanded}
+				collapsible={collapsible}
```

Comment:
- schema supports `data-testid` so `id` should not be applied to it
- can spread `otherSchema` here instead of applying every prop manually